lang/ilias_de.lang

```diff
@@ -7111,6 +7111,7 @@ mail#:#mail_email_sys_subject#:#Test-E-Mail „System-Mail”
 mail#:#mail_email_sys_body#:#Dies ist eine Test-E-Mail um den Versand einer automatisch vom System initiierten E-Mail an externe E-Mail-Adressen zu testen.
 mail#:#mail_is_read#:#gelesen
 mail#:#mail_is_unread#:#ungelesen
+mail#:#reg_mail_user_activation_status#:#Aktivierungsstatus des Nutzeraccounts
```

Comment:
```suggestion
mail#:#reg_mail_user_activation_status#:#Aktivierungsstatus des Benutzerkontos
```

Reply:
I have just added the English lang file. Committing correctly is not my strong side, it seems ;)